cmd/root.go

```diff
@@ -123,6 +123,13 @@ func init() {
 	viper.BindPFlag("skip-workspace-crud",
 		rootCmd.PersistentFlags().Lookup("skip-workspace-crud"))
 
+	// Support for Session Cookie
+	rootCmd.PersistentFlags().String("kong-session-cookie-path", "",
```

Comment:
```suggestion
	rootCmd.PersistentFlags().String("kong-cookie-jar-path", "",
```

Let's avoid the word 'session' in here since the functionality is more generic than that.

Reply:
Done